src/Synolia/Bundle/StockAlertBundle/Twig/QuantityExtension.php

```diff
@@ -51,27 +43,30 @@ public function getFunctions()
         ];
     }
 
-    public function getProductQuantity($product): int
+    public function getProductQuantity(mixed $product): int
```

Comment:
mixed D:< 

now we can add the possible types like:

`public function getProductQuantity(array|Product $product): int`

Reply:
well... mixed is also enough I guess xD

Reply:
i'm not sure but there are possibly 3 types you can pass via template imports ; `Product|ProductView|array` so it's very confusing